### PR TITLE
Fix testing matrix key for Node.js version

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
       # TODO: Split the linting into a separate job.
       - name: Run lint task
         # Only run this step for the default Node version.
-        if: matrix.node == env.DEFAULT_NODE_VERSION
+        if: matrix.node-version == env.DEFAULT_NODE_VERSION
```
Comment (mabartos) [inline .github/workflows/main.yml:40]: You don't need here `${{ matrix.node-version }} == ....` ?

Comment (jonkoops) [reply .github/workflows/main.yml:40]: No, everything inside of `if` is considered an expression so there is no need to open an expression here manually.

Comment (mabartos) [reply .github/workflows/main.yml:40]: Ok, thanks for the explanation. I was just curious about it.

**Review by mabartos — APPROVED** [review]
Approved; even if the CI is broken. It could be done in some follow-up task.

**Review by abstractj — DISMISSED** [review]

**Pull request merged**